Dockerfile

```diff
@@ -7,9 +7,9 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get -y update && \
```

Comment:
I wish we 1. moved wget up there and 2. organized the package list alphabetically, but that can wait for a separate PR. I have a separate Dockerfile which is tidied up a little, which I'm using for my windows box.